java-checks/src/main/java/org/sonar/java/checks/MethodOnlyCallsSuperCheck.java

```diff
@@ -46,8 +46,7 @@
 @Rule(key = "S1185")
 public class MethodOnlyCallsSuperCheck extends IssuableSubscriptionVisitor {
 
-  private static final List<String> TRANSACTIONAL_ANNOTATIONS = Arrays.asList("javax.transaction.Transactional",
-    "org.springframework.transaction.annotation.Transactional");
+  private static final List<String> TRANSACTIONAL_ANNOTATIONS = Arrays.asList("javax.transaction.Transactional");
```

Comment:
```suggestion
  
```
We could remove this constant and instead at line 169 have a simple 
`return metadata.isAnnotatedWith("javax.transaction.Transactional");`